var/spack/repos/builtin/packages/lvarray/package.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import warnings
+import os
+
+
+class Lvarray(CMakePackage, CudaPackage):
+    """LvArray portable HPC containers."""
+
+    homepage = 'https://github.com/GEOSX/lvarray'
+    git      = 'https://github.com/GEOSX/LvArray.git'
+
+    version('develop', branch='develop', submodules='True')
+    version('tribol', branch='feature/corbett/tribol', submodules='True')
+
+    variant('shared', default=True, description='Build Shared Libs')
+    variant('umpire', default=False, description='Build Umpire support')
+    variant('chai', default=False, description='Build Chai support')
+    variant('caliper', default=False, description='Build Caliper support')
+    variant('tests', default=True, description='Build tests')
+    variant('benchmarks', default=False, description='Build benchmarks')
+    variant('examples', default=False, description='Build examples')
+    variant('docs', default=False, description='Build docs')
+    variant('addr2line', default=True, description='Build support for addr2line.')
+
+    depends_on('cmake@3.8:', type='build')
+    depends_on('cmake@3.9:', when='+cuda', type='build')
+
+    depends_on('raja')
+    depends_on('raja+cuda', when='+cuda')
+
+    depends_on('umpire', when='+umpire')
+    depends_on('umpire+cuda', when='+umpire+cuda')
+
+    depends_on('chai+raja', when='+chai')
+    depends_on('chai+raja+cuda', when='+chai+cuda')
+
+    depends_on('caliper', when='+caliper')
+
+    depends_on('doxygen@1.8.13:', when='+docs')
+    depends_on('py-sphinx@1.6.3:', when='+docs')
```

Comment:
Probably `type='build'`

Reply:
As a keyword argument?
```
depends_on('doxygen@1.8.13', when'+docs', type='build')`? 
```
And would this only add the dependency when executing the build stage?

Reply:
Yes, that's the correct syntax. The dependency would only be added to the `PATH` and `PYTHONPATH` during installation, it wouldn't be linked to using Spack's compiler wrappers or affect the hash in any way.

Reply:
Nice, done.